src/ui.js

```diff
@@ -122,6 +122,7 @@ function createOptionsPanel() {
   elmContainer.appendChild(createConfigCheckbox('enableAdBlock'));
   elmContainer.appendChild(createConfigCheckbox('hideLogo'));
   elmContainer.appendChild(createConfigCheckbox('enableSponsorBlock'));
+  elmContainer.appendChild(createConfigCheckbox('removeShorts'));
```

Comment:
This should probably either be at the end or before `enableSponsorBlock`. (Same in `config.js`.)